### ISSUE TITLE
Karabiner stops working? M1

### ISSUE THREAD

**Issue by LegendreJ** [pqrs-org/Karabiner-Elements#2796]
I use karabiner to remap some keys ONLY when playing DOTA2 using this script:

`{
  "title": "Swap Left Option/Command only in Dota2",
  "rules": [
    {
      "description": "Left Option to Command",
      "manipulators": [
        {
          "type": "basic",
          "from": {
            "key_code": "left_option"
          },
          "to": [
            {
              "key_code": "left_command"
            }
          ],
          "conditions": [
            {
              "type": "frontmost_application_if",
              "bundle_identifiers": [
                "^com.valvesoftware.dota2"
              ]
            }
          ]
        }
      ]
    },
    {
      "description": "Left Command to Option",
      "manipulators": [
        {
          "type": "basic",
          "from": {
            "key_code": "left_command"
          },
          "to": [
            {
              "key_code": "left_option"
            }
          ],
          "conditions": [
            {
              "type": "frontmost_application_if",
              "bundle_identifiers": [
                "^com.valvesoftware.dota2"
              ]
            }
          ]
        }
      ]
    }
  ]
}`

It works fine and does what it needs to do but sometimes after playing a couple games consecutively it seems to just stop working. The remap doesn't work any more and the only way I can get it to work again is to restart the laptop.

Details:
MacBook Air M1 8gpu/16gb/512sss
macOS Big Sur 11.5.1

**Comment by lahariganti** [comment]
Yeah, running 11.5.2 (20G95) on an MacBook Pro (13-inch, M1, 2020); basic modifiers don't seem to . . . work? Have re-installed and restarted multiple times now

**Comment by LegendreJ** [comment]
I'm just responding to this again because the issue is happening much more frequently now. However, if i open Karabiner>Misc>Restart Karabiner Elements, it solves the issue temporarily.

Karabiner v14.2.0
macOS Montery 12.0.1
MacBook Air M1 8gpu/16gb/512sss

**Comment by LegendreJ** [comment]
I'm on Karabiner V14.5.0 and i still have this issue 

:(
